src/networkMaps/create/utils/__tests__/buildNetworkMappings.test.ts

```diff
@@ -36,7 +36,7 @@ describe('buildNetworkMappings', () => {
       const mappings = [
         {
           ...mockNetworkMapping,
-          [NetworkMapFieldId.TargetNetwork]: { name: 'Pod network' },
+          [NetworkMapFieldId.TargetNetwork]: { name: 'Default network' },
```

Comment:
Please replace 'Default network' with [DEFAULT_NETWORK](https://github.com/kubev2v/forklift-console-plugin/blob/ecb217499a68acf00984c8968538a7b7c7589ef2/src/utils/constants.ts#L58) const for all occurrences

Reply:
For tests, this isn't important to use all constants, but sure, I can do that.